cime_config/config_grids.xml

```diff
@@ -3189,6 +3295,22 @@
       <map name="OCN2ATM_SMAPNAME">cpl/gridmaps/ECwISC30to60E2r1/map_ECwISC30to60E2r1-nomask_to_ne30pg2_mono.201006.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne30np4.pg2" ocn_grid="ECwISC30to60E3r2">
+      <map name="ATM2OCN_FMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_ECwISC30to60E3r2_mono.230901.nc</map>
+      <map name="ATM2OCN_VMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_ECwISC30to60E3r2_bilin.230901.nc</map>
+      <map name="ATM2OCN_SMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_ECwISC30to60E3r2-nomask_bilin.230901.nc</map>
```

Comment:
Was this made with TempestRemap?

Reply:
@jonbob, that's a question for you.

Reply:
We've been making all conservative "mono" mapping files with tempest, but not the bilinear ones. I'll try to change the process to match the new naming convention and process